dfttopif/parsers/pwscf.py

```diff
@@ -15,6 +16,25 @@ def __init__(self, directory):
         super(PwscfParser, self).__init__(directory)
         self.settings = {}
         parser = PwscfStdOutputParser()
+
+        # Look for appropriate files
+        try:
+            self.inputf = self.outputf = None
+            files = [f for f in os.listdir(self._directory) if os.path.isfile(os.path.join(directory, f))]
+            for f in files:
+                if self._get_line('Program PWSCF', f, basedir=self._directory, return_string=False):
+                    self.outputf = f
+                elif self._get_line('&control', f, basedir=self._directory, return_string=False, case_sens=False):
+                    self.inputf = f
+        except Exception as e:
+            raise InvalidIngesterException('Parser failed due to: ' + str(e))
+        print(self.inputf, self.outputf)
```

Comment:
Clean this out or replace it with a logger.

Reply:
Oops! I thought I already killed that line (used when debugging)